client/components/upgrades/gsuite/gsuite-upsell-card/index.jsx

```diff
@@ -85,11 +87,15 @@ const GSuiteUpsellCard = ( {
 			<CompactCard>
 				<header className="gsuite-upsell-card__header">
 					<h2 className="gsuite-upsell-card__title">
-						{ translate( 'Add professional email from G Suite by Google Cloud to %(domain)s', {
-							args: {
-								domain,
-							},
-						} ) }
+						{ translate(
+							'Add professional email from %(serviceName)s by Google Cloud to %(domain)s',
```

Comment:
I think the `by Google Cloud` part should also go. This tends to be also consistent with #49083